### PR TITLE
Structure with basic styles

### PR DESCRIPTION
**Pull request by andreabettich**
Basis for the new design with gulp sass task with browsersync.
`npm install && npm start`

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+  <head>
+    <meta charset="utf-8">
+    <title>Freedomvote</title>
+    <link rel="stylesheet" href="main.css">
+  </head>
+  <body>
+    <header class="navigation">
+      <nav class="navigation__nav">
+        <a href="#">
+          <img class="navigation__nav__logo" src="/img/logo.svg" />
```
Comment (kennyzehn) [inline public/index.html]: Dont nest to much. `.navigation__nav__logo` is okay but not necessary. I would prefer `.navigation__logo`

File: public/index.html [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+        </a>
+        <ul class="navigation__nav__page">
+          <li><a href="#" class="navigation__nav__page__active">Home</a></li>
```
Comment (kennyzehn) [inline public/index.html]: Use a modifier to show the active status.

File: public/index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+      </nav>
+    </header>
+    <div class="container">
+      <h1 class="hero">Freiheit und Sicherheit war noch nie einfach</h1>
+      <h2 class="headline">Freiheit und Sicherheit war noch nie einfach</h1>
+      <h2 class="headline-fancy">Freiheit und Sicherheit war noch nie einfach</h1>
+
+      <p>Besonders jetzt im Informationszeitalter wird das Thema immer schwieriger und komplexer. Freiheit ermöglicht uns einen innovativen Umgang mit Technologie, schafft Zugang zu Wissen und fördert Kreativität. Der Staat ist jedoch auch in der Verantwortung für Sicherheit zu sorgen, damit diejenigen zur Rechenschaft gezogen werden können, welche die ermöglichten Freiheiten missbrauchen.</p>
+      <div class="boxes">
+        <div class="box deep-lavender"></div>
```
Comment (kennyzehn) [inline public/index.html]: Use a modifiere. `.box .box--primary`

File: public/index.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+        <a href="#">
+          <img class="navigation__nav__logo" src="/img/logo.svg" />
+        </a>
+        <ul class="navigation__nav__page">
+          <li><a href="#" class="navigation__nav__page__active">Home</a></li>
+          <li><a href="#">Kandidaten</a></li>
+          <li><a href="#">Vergleichen</a></li>
+          <li><a href="#">Partner</a></li>
+        </ul>
+        <i class="navigation__nav__divider"></i>
```
Comment (kennyzehn) [inline public/index.html]: Dont nest to much. `.navigation__nav__divider` is okay but not necessary. I would prefer `.navigation__divider`

File: public/index.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+        <nav class="navigation__nav__language">
```
Comment (kennyzehn) [inline public/index.html]: Create a own component for this element. So that you can take it and place anywhere else.

File: public/index.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        <i class="navigation__nav__search">Search</i>
+      </nav>
+    </header>
+    <div class="container">
+      <h1 class="hero">Freiheit und Sicherheit war noch nie einfach</h1>
+      <h2 class="headline">Freiheit und Sicherheit war noch nie einfach</h1>
+      <h2 class="headline-fancy">Freiheit und Sicherheit war noch nie einfach</h1>
+
+      <p>Besonders jetzt im Informationszeitalter wird das Thema immer schwieriger und komplexer. Freiheit ermöglicht uns einen innovativen Umgang mit Technologie, schafft Zugang zu Wissen und fördert Kreativität. Der Staat ist jedoch auch in der Verantwortung für Sicherheit zu sorgen, damit diejenigen zur Rechenschaft gezogen werden können, welche die ermöglichten Freiheiten missbrauchen.</p>
+      <div class="boxes">
```
Comment (kennyzehn) [inline public/index.html]: Creeate a BEM Component. For Example:
`<div class="box"><div class="box__item"></div></div>"`

My naming is bad… im sure you can find a better name :P

File: src/sass/components/_box.sass [hunk 7]
```diff
@@ -0,0 +1,18 @@
+.box
+  width: 175px
+  height: 175px
+  display: inline-block
+  &.deep-lavender
```
Comment (kennyzehn) [inline src/sass/components/_box.sass]: Create a BEM modifiere (`.box--white`)

File: src/sass/components/_box.sass [hunk 8]
```diff
@@ -0,0 +1,18 @@
...
+    background-color: $deep-lavender
+  &.white
```
Comment (kennyzehn) [inline src/sass/components/_box.sass]: Don´t use colors as className.

File: src/sass/components/_sections.sass [hunk 9]
```diff
@@ -0,0 +1,7 @@
+.section
+  padding: 100px 320px
+  background-color: $white
+  text-align: center
+  &-fancy
```
Comment (kennyzehn) [inline src/sass/components/_sections.sass]: Create a BEM Modifiere by using two dashes (`&--fancy`)

File: src/sass/layout/_navigation.sass [hunk 10]
```diff
@@ -0,0 +1,27 @@
+.navigation
```
Comment (kennyzehn) [inline src/sass/layout/_navigation.sass:7]: Try to keep it simple. Don´t nest to much. Best practice are two levels deep. In worst case three levels deep.

File: src/sass/themes/_base.sass [hunk 11]
```diff
@@ -0,0 +1,10 @@
+
+html, body
+  height: 100%
+
+body
```
Comment (kennyzehn) [inline src/sass/themes/_base.sass]: Is this line necessary?

File: public/img/logo.svg [hunk 12]
```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```
Comment (kennyzehn) [inline public/img/logo.svg]: In SVG files are a lot of crap. if you like you can remove the most of them. 
* All IDs
* All Meta descriptions
* the version
* <defs />

Comment (andreabettich) [reply public/img/logo.svg]: Done

**Comment by andreabettich** [comment]
@kennyzehn can you review again?

File: src/sass/layout/_navigation.sass [hunk 13]
```diff
@@ -0,0 +1,47 @@
...
+  background-color: $navigation-bgcolor
+
+.navigation__nav
+    padding: 37px 100px
+    height: 32px
+    display: flex
+    flex-direction: row
+    align-items: center
+
+.navigation .menu
```
Comment (kennyzehn) [inline src/sass/layout/_navigation.sass]: Is the first selector necessary?

File: src/sass/layout/_navigation.sass [hunk 14]
```diff
@@ -0,0 +1,47 @@
...
+  &__item
+    a
+      +font-droid(17px, $menu-item-color, normal, 0.6px, 0px)
+      text-decoration: none
+      padding: 0 1rem
+    &--active
+      a
+        font-weight: 600
+
+.navigation__logo
```
Comment (kennyzehn) [inline src/sass/layout/_navigation.sass]: Maybe its cleaner to you use the `&__`  SASS-Feature?

File: src/sass/layout/_navigation.sass [hunk 15]
```diff
@@ -0,0 +1,47 @@
...
+  width: 250px
+
+.navigation__divider
+  height: 22px
+  border: solid 1px $navigation-divider-color
+
+.navigation__language
```
Comment (kennyzehn) [inline src/sass/layout/_navigation.sass]: Maybe its cleaner to you use the `&__`  SASS-Feature?

File: src/sass/layout/_navigation.sass [hunk 16]
```diff
@@ -0,0 +1,47 @@
...
+  .language
+    list-style: none
+  .language__item
```
Comment (kennyzehn) [inline src/sass/layout/_navigation.sass]: Maybe its cleaner to you use the `&__`  SASS-Feature?

File: src/sass/layout/_navigation.sass [hunk 17]
```diff
@@ -0,0 +1,47 @@
...
+    color: $language-item-color
+    display: none
+    &--active
+      display: block
+
+.navigation__search
```
Comment (kennyzehn) [inline src/sass/layout/_navigation.sass]: Maybe its cleaner to you use the `&__`  SASS-Feature?

**Comment by kennyzehn** [comment]
Looks good!

**Pull request merged**